sdk/storage/azure-storage-queue/azure/storage/queue/aio/_queue_service_client_async.py

```diff
@@ -92,19 +92,18 @@ def __init__(
         ) -> None:
         kwargs['retry_policy'] = kwargs.get('retry_policy') or ExponentialRetry(**kwargs)
         loop = kwargs.pop('loop', None)
-        super(QueueServiceClient, self).__init__( # type: ignore
+        super(QueueServiceClient, self).__init__(
             account_url,
             credential=credential,
             loop=loop,
             **kwargs)
-        self._client = AzureQueueStorage(self.url, base_url=self.url, pipeline=self._pipeline, loop=loop) # type: ignore
+        self._client = AzureQueueStorage(self.url, base_url=self.url, pipeline=self._pipeline, loop=loop)
         self._client._config.version = get_api_version(kwargs)  # pylint: disable=protected-access
         self._loop = loop
         self._configure_encryption(kwargs)
 
     @distributed_trace_async
-    async def get_service_stats(self, **kwargs):
-        # type: (Optional[Any]) -> Dict[str, Any]
+    async def get_service_stats(self, **kwargs: Any) -> Dict[str, Any]:
```

Comment:
Also want to point this out -- we generally use just `Any` for `**kwargs`, but here the original typehint was `Optional[Any]`. Just want to get consensus and then will apply for this file and all future files